subsys/bluetooth/controller/Kconfig

```diff
@@ -310,5 +311,53 @@ config BT_UNINIT_MPSL_ON_DISABLE
 	  mpsl_lib_uninit, which releases all peripherals and allows the user to
 	  override any interrupt handlers used by MPSL.
 
+config BT_CTLR_SDC_PAWR_ADV
+	bool "LE Periodic Advertising with Responses Advertising State [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	select BT_CTLR_CHAN_SEL_2
+	depends on BT_PER_ADV
+	depends on BT_CTLR_SYNC_TRANSFER_SENDER
+	help
+	  Enable support for Periodic Advertising with Respsonses - Advertiser.
+
+if BT_CTLR_SDC_PAWR_ADV
+config BT_CTLR_SDC_PAWR_ADV_COUNT
+	int "Periodic Advertising with Responses - Advertiser sets"
+	range 0 10
+	default 1
+	help
+	  Maximum number of concurrent Periodic Advertising with Responses
+	  advertisers supported.
+
+config BT_CTLR_SDC_PERIODIC_ADV_RSP_TX_BUFFER_COUNT
+	int "Number of PAwR advertiser TX buffers"
+	range 0 15
```

Comment:
```suggestion
	range 1 15
```

0 isn't allowed

Reply:
why 15? any theory

Reply:
What if the host uses multiple commands? The count parameter of data requests is limited to the number of subevents, so max 128 -- I'd imagine someone could want to allocate more tx buffers to reduce the amount of traffic over HCI

Reply:
Can move the upper limit to 128. I guess the device will run out of memory before it hits those limits anyway.